src/v3/src/util/theme.ts

```diff
@@ -114,13 +113,12 @@ export const generatePalette = (main: string): Palette => {
 
 export const createTheme = (
   brandColors?: BrandColors,
-  customTokens?: Partial<DesignTokensType>,
+  customTokens?: Partial<DesignTokensOverride>,
 ): ThemeOptions => {
-  const defaultTokens = DESIGN_TOKENS;
-  const mergedTokens: DesignTokensType = { ...defaultTokens, ...customTokens };
-  const theme = odysseyTheme;
+  // Odyssey 1.x does not export their theme, but we can recreate it
+  const theme = createOdysseyMuiTheme({ odysseyTokens: { ...Tokens, ...customTokens } });
```

Comment:
Here is where we have to create our own version of the OD 1.x theme and pass in our token overrides